README.ipynb

```diff
@@ -24,6 +24,14 @@
     "  * `~/work/outputs/outputs_4`\n",
     "* **Do  not write files directly to outputs:** if your code generates files, we recommend writing them first to a temporary directory (e.g. in `~/work/workspace`) and then moving them to the desired `~/work/outputs/output_X` directory after completion. Otherwise, data synchronisation may start while the file is being written to, which may result in a corrupted file in output.\n",
     "\n",
+    "#### Dependency installation\n",
+    "**To correctly install a python dependency,** for example `numpy`, in a new code cell inside the notebook use the following\n",
+    "```python\n",
+    "import sys\n",
+    "!{sys.executable} -m pip install numpy\n",
+    "```\n",
+    "For more details see [Installing Python Packages from a Jupyter Notebook](https://jakevdp.github.io/blog/2017/12/05/installing-python-packages-from-jupyter/#How-to-use-Pip-from-the-Jupyter-Notebook).\n",
+    "\n",
     "#### Voila mode\n",
     "\n",
     "This notebook supports boot as voila mode. To start a notebook as voila, set the `JupyterLab` to Boot in `Voila` in the platform\n",
```

Comment:
In "Have an issue?" section, I guess we want them to be opened in this repository (at https://github.com/ITISFoundation/jupyter-math/issues)